modules/service/src/main/java/io/oasp/module/service/common/impl/header/ServiceHeaderCustomizerJwt.java

```diff
@@ -0,0 +1,46 @@
+package io.oasp.module.service.common.impl.header;
+
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+import io.oasp.module.service.common.api.config.ServiceConfig;
+import io.oasp.module.service.common.api.header.ServiceHeaderContext;
+import io.oasp.module.service.common.api.header.ServiceHeaderCustomizer;
+
+/**
+ * @author ssarmoka
+ *
+ */
+public class ServiceHeaderCustomizerJwt implements ServiceHeaderCustomizer {
+
+  private static final String AUTHORIZATION = "Authorization";
+
+  private static final String CONTENT_TYPE = "Content-Type";
+
+  /**
+   *
+   * The constructor.
+   */
+  public ServiceHeaderCustomizerJwt() {
+    super();
+  }
+
+  @Override
+  public void addHeaders(ServiceHeaderContext<?> context) {
+
+    String auth = context.getConfig().getChildValue(ServiceConfig.KEY_SEGMENT_AUTH);
+    if (!"jwt".equals(auth)) {
+      return;
+    }
+    SecurityContext securityContext = SecurityContextHolder.getContext();
+    if (securityContext == null) {
+      return;
+    }
+
+    String authorizationHeader = context.getConfig().getChildValue(AUTHORIZATION);
+    String contentType = context.getConfig().getChildValue(CONTENT_TYPE);
+    context.setHeader(AUTHORIZATION, authorizationHeader + "");
+    context.setHeader(CONTENT_TYPE, contentType);
```

Comment:
I would either remove the content type entirely or if really needed move it to another implementation. I still do not get how this is related to security/authentication or what this is about at all.